cpplint.py

```diff
@@ -4744,8 +4859,10 @@ def _ClassifyInclude(fileinfo, include, is_system):
   if is_system:
     if is_cpp_h:
       return _CPP_SYS_HEADER
-    else:
+    if is_c_h:
       return _C_SYS_HEADER
+    else:
+      return _OTHER_HEADER
```

Comment:
not sure about this part, so this would require any includes done with <> to come last in the list of headers? Maybe a separate category _OTHER_SYSTEM_HEADER would be more consistent with user expectations? I am wary of breaking current clients with this, but not sure how often this happens...

Reply:
I added a new category `_OTHER_SYS_HEADER`